modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -1746,7 +1745,7 @@ private CompletableFuture<FuturesCleanupResult> awaitCleanupReadyFutures(UUID tx
 
             txOps.futures.clear();
 
-            return txOps;
+            return null;
```

Comment:
Do we use cleanupReadyFutures in order to handle the race between cleanup and adding new operations? Meaning that: 

- we need to await all pending operations prior to cleanup processing - all good here.
- we need to discard new operation after cleanup was done - do we use cleanupReadyFutures to handle such race?

Reply:
Hmm, the question is still on the table.